### PR TITLE
vendor: Update virtcontainers vendoring for vcMock

### LINKED ISSUE #447: Update virtcontainers vendoring for vcMock
Update the vendoring to be able to make use of the `vcMock` framework (https://github.com/containers/virtcontainers/pull/348).

### PR DESCRIPTION
**Pull request by jodh-intel**
Shortlog of what have been added since the last vendoring:

    61650fe api/tests: Introduce a test mock

Fixes #447

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: start.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ var startCLICommand = cli.Command{
 	},
 }
 
-func start(containerID string) (*vc.Pod, error) {
+func start(containerID string) (vc.VCPod, error) {
```
Comment (grahamwhaley) [inline start.go]: presume this was to match a change in the updated vendor code base?

**Comment by grahamwhaley** [comment]
lgtm

**Comment by sboeuf** [comment]
LGTM

File: start.go (hunk 1 above)
Comment (jodh-intel) [inline start.go]: yes, won't build without it.

**Pull request merged**